<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ANTWidget</class>
 <widget class="QWidget" name="ANTWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base file name</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Message</string>
   </property>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessageLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Message to send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MessageEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">00-00-00-00-00-00-00-00</string>
    </property>
    <property name="text">
     <string>00-00-00-00-00-00-00-00</string>
    </property>
   </widget>
   <zorder>MessageLabel</zorder>
   <zorder>SendButton</zorder>
   <zorder>MessageEdit</zorder>
  </widget>
  <widget class="QLabel" name="USBDeviceNumberLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>USB port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisconnectButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>ANT Configuration</string>
   </property>
   <widget class="QLineEdit" name="RFEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>66</string>
    </property>
   </widget>
   <widget class="QLabel" name="HzLabel">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>72</y>
      <width>125</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2400Mhz+         Mhz</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeviceTypeLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>132</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="TransmissionTypeLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>162</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transmission Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeviceNumberLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>101</y>
      <width>94</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChannelTypeLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>41</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="RFLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>71</y>
      <width>102</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio Frequency</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ChannelTypeComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="currentText">
     <string>Slave (channel 0 only)</string>
    </property>
    <item>
     <property name="text">
      <string>Slave (channel 0 only)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slave Multi Channel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Master</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="DeviceNumberEdit">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>97</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TransmissionTypeEdit">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>159</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DeviceTypeEdit">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>128</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>DeviceTypeLabel</zorder>
   <zorder>TransmissionTypeLabel</zorder>
   <zorder>DeviceNumberLabel</zorder>
   <zorder>ChannelTypeLabel</zorder>
   <zorder>RFLabel</zorder>
   <zorder>ChannelTypeComboBox</zorder>
   <zorder>DeviceNumberEdit</zorder>
   <zorder>TransmissionTypeEdit</zorder>
   <zorder>DeviceTypeEdit</zorder>
   <zorder>HzLabel</zorder>
   <zorder>RFEdit</zorder>
  </widget>
  <widget class="QPushButton" name="NewDevice">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="USBDeviceNumberEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>490</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>File</string>
   </property>
   <widget class="QLineEdit" name="BaseFileNameEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>myResult</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileDirectoryLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>174</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="StartDistanceEdit">
       <property name="text">
        <string>20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="IncrementLabel">
       <property name="text">
        <string>Incr.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IncrementEdits">
       <property name="text">
        <string>20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>211</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of measurements</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>count</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="NumOfMeasurementsEdit">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CountEdit">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>USB Configuration</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
